packages/ckeditor5-font/src/fontcolor/fontcolorediting.js

```diff
@@ -115,6 +115,20 @@ export default class FontColorEditing extends Plugin {
 			}
 		} );
 
+		// Support legacy `<font color="..">` formatting.
+		editor.conversion.for( 'upcast' ).elementToAttribute( {
+			view: {
+				name: 'font',
+				attributes: {
+					'color': /^#?\w+$/
+				}
+			},
+			model: {
+				key: FONT_COLOR,
+				value: viewElement => viewElement.getAttribute( 'color' )
```

Comment:
For the `AABBCC` format that would not be a valid color but it is handled properly by the browsers in the `<font>` tag but not in CSS. But I think that information won't be lost if we don't fix it (it's still in the data but in invalid format).